### DISCUSSION TITLE
Synthea Databricks

### DISCUSSION THREAD

**Discussion by tariqsherwani1** [synthetichealth/synthea#1228] [Q&A] [answered]
Hi,

Is there a way to directly run Synthea in Databricks? I need to generate around 500k records.

Thanks

**Comment by hadleynet** [comment, marked as answer]
Looks like you could run the [Synthea Jar file](https://github.com/synthetichealth/synthea/wiki/Basic-Setup-and-Running) in a [Databricks job](https://docs.databricks.com/workflows/jobs/how-to-use-jars-in-workflows.html).

If you get it work, we'd appreciate a write up on how to do it that we could add to the wiki or FAQ.
